pom.xml

```diff
@@ -108,6 +108,11 @@
             <!--TODO: change the snapshot version to 1.0.0 -->
             <version>0.9.1-SNAPSHOT</version>
         </dependency>
+        <dependency>
```

Comment:
This is a difficult dependency to manage and easily conflicts with what the customer is using. I would look for anyway to avoid it.